executor/insert_test.go

```diff
@@ -1131,75 +702,6 @@ func TestInsertFloatOverflow(t *testing.T) {
 	tk.MustExec("drop table if exists t,t1")
 }
 
-// TestAutoIDIncrementAndOffset There is a potential issue in MySQL: when the value of auto_increment_offset is greater
-// than that of auto_increment_increment, the value of auto_increment_offset is ignored
-// (https://dev.mysql.com/doc/refman/8.0/en/replication-options-master.html#sysvar_auto_increment_increment),
-// This issue is a flaw of the implementation of MySQL and it doesn't exist in TiDB.
-func TestAutoIDIncrementAndOffset(t *testing.T) {
```

Comment:
Updated and moved to https://github.com/pingcap/tidb/pull/38808/files#diff-1c27a2956e8d617d4597580c00fb81d42bf81a4a7e726d911e11ac25ccb1e484R594